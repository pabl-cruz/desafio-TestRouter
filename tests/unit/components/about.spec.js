import { shallowMount } from '@vue/test-utils'

import About from '@/components/About.vue'

describe('AboutView', () => {
    it('captura de contenido HTML en prueba', () => {
        const wrapper = shallowMount(About)
        expect(wrapper.html()).toMatchSnapshot()
    })
})