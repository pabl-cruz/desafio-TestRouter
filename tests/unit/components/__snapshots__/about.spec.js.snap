// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`AboutView captura de contenido HTML en prueba 1`] = `""`;
